src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAgreement/WhenDoYouWantToView.cshtml

```diff
@@ -105,106 +105,3 @@
       });
     </script>
 }
-
-
-
-@*<div class="govuk-grid-row">
```

Comment:
what is all this commented code?

Reply:
From the GitHub history it looks like it was commented out whilst the CDN work was being done in 2019 and not removed - so I've junked it now...